tests/integration/test.aa.setup.js

```diff
@@ -10,4 +10,9 @@ describe('DB Setup', function () {
       }
     );
   });
+
+  it('PouchDB has a version', function () {
+    PouchDB.version.should.be.a('string');
+    PouchDB.version.should.match(/\d+\.\d+\.\d+/);
+  });
```

Comment:
I also went ahead and confirmed manually that the version is currently `'5.1.1-prerelease'`.